processing/src/test/java/io/druid/query/filter/BetweenDimFilterTests.java

```diff
@@ -0,0 +1,53 @@
+package io.druid.query.filter;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Injector;
+import com.google.inject.Key;
+import io.druid.guice.GuiceInjectors;
+import io.druid.guice.annotations.Json;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+@RunWith(Parameterized.class)
+public class BetweenDimFilterTests
+{
+  public BetweenDimFilterTests(BetweenDimFilter betweenDimFilter) {this.betweenDimFilter = betweenDimFilter;}
+
+  private final BetweenDimFilter betweenDimFilter;
+
+  @Parameterized.Parameters
+  public static Iterable<Object[]>  constructorFeeder(){
+
+    return ImmutableList.of(new Object[]{new BetweenDimFilter("dimension", "12", "15", null, null, false)},
```

Comment:
@drcrallen i can add all the combinations, but for the purpose knowledge  i don't get why we need all the 8 combinations ? do you have a corner case in mind ?